examples/notebooks/Client_Side_Processing/client_side_processing_color_composites.ipynb

```diff
@@ -0,0 +1,443 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3cfbd63f-f9fe-4521-af0a-5679216871df",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "<img src=\"https://avatars.githubusercontent.com/u/74911464?s=200&v=4\"\n",
+    "     alt=\"OpenEO Platform logo\"\n",
+    "     style=\"float: left; margin-right: 10px;\" />\n",
+    "# OpenEO Platform - Client Side Processing\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "739a4107-cc33-4948-ae72-74e41efc5128",
+   "metadata": {},
+   "source": [
+    "In this interactive notebook we will show some usage eaxample of the Client Side Processing functionality added recently to the openEO Python client."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "827ec0c9-c38f-4d2f-a293-00d110ec70f6",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Requirements"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51a9e53a-481c-4ea5-a175-652d6398828d",
+   "metadata": {},
+   "source": [
+    "To use this functionality, you need `3.9<=Python<3.11`.\n",
```

Comment:
Comes from here: https://github.com/Open-EO/openeo-processes-dask/blob/6563c1809fc4108581b78763a970b3bea1d5d935/pyproject.toml#L25

@LukeWeidenwalker how could I help to test more Python versions? It would be good to add 3.11 at least.

Reply:
When I initially set up the repo, I put 3.9, 3.10 and 3.11 in the CI matrix for github actions. 3.11 failed for a reason that I can't recall now, and since our backend runs on 3.9, I've just ignored it since. Would be in favor of becoming 3.11 compatible though! I'd start by including 3.11 in the matrix here: https://github.com/Open-EO/openeo-processes-dask/blob/main/.github/workflows/main.yml in a new PR and just seeing what breaks. Once the build passes, we can remove this constraint!